app/experimenter/templates/experiments/list.html

```diff
@@ -25,16 +25,20 @@ <h3 class="m-0">
       {{ filter.form.firefox_version.value }}
     {% endif %}
 
-    {% if filter.form.in_qa.value == "in_qa" %}
-       in QA
-    {% endif %}
-
     {% if filter.form.project.value %}
       {{ filter.form.get_project_display_value }}
     {% endif %}
 
     Experiment{{ paginator.count|pluralize:"s" }}
 
+    {% if filter.form.in_qa.value %}
+       in QA
+    {% endif %}
+
```

Comment:
While I was adding the title language for surveys, I realized that this piece of code right here, introduced by me weeks back, was in a non-ideal location. With it being in it's previous location, above the code that inserts the word "Experiment" into the title at line 32, the title reads " X in QA Experiments" instead of the better title... "X Experiments in QA". That's why I moved this bit of code to below line 32.

Reply:
Ah the ol PR switcheroo